components/list/list-item-placement-marker.js

```diff
@@ -0,0 +1,90 @@
+import '../colors/colors.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class ListItemPlacementMarker extends RtlMixin(LitElement) {
+	static get properties() {
+		return {
+			role: {type: String, reflect: true}
+		};
+	}
+
+	static get styles() {
+		return css`
+			:host {
+				display: block;
+			}
+
+			:host([hidden]) {
+				display: none;
+			}
+
+			.d2l-list-drag-marker-line {
+				height: 12px;
+				width: 100%;
+				stroke: var(--d2l-color-celestine);
+				stroke-width: 3px;
+				margin-left: -1px;
+				margin-right: -1px;
+			}
+
+			.d2l-list-drag-marker-linecap {
+				height: 12px;
+				width: 4px;
+				stroke: none;
+				fill: var(--d2l-color-celestine);
+				margin-left: -1px;
+				margin-right: 0;
+			}
+
+			:host([dir="rtl"]) .d2l-list-drag-marker-linecap {
+				margin-left: 0;
+				margin-right: -1px;
+			}
+
+			.d2l-list-drag-marker-circle {
+				height: 12px;
+				width: 12px;
+				stroke-width: 3px;
+				stroke: var(--d2l-color-celestine);
+				fill: none;
+				margin-left: 0;
+				margin-right: -1px;
+			}
+
+			:host([dir="rtl"]) .d2l-list-drag-marker-circle {
+				margin-left: -1px;
+				margin-right: 0;
+			}
+
+			.d2l-list-drag-marker {
+				flex-wrap: nowrap;
+				display: flex;
+			}
+		`;
+	}
+
+	constructor() {
+		super();
+		this.role = 'img';
```

Comment:
This is interesting.  What's the rationale for marking this up to be an image, semantically?  What does the screen reader communicate?  I think this is the first time I've seen `img` being assigned for the role.

Reply:
@m6jones  could you please comment on dave's question ?

Reply:
So to me this is an image component. So I wanted to express it as an image. Secondary, I want people to be able to define this as decorative or have alt text if their use case needs it.

Reply:
My thinking is that this would always be decorative.  In what case would we want a consumer of this component be able to define it as decorative or not?  How would this get surfaced to AT and would it be meaningful as an image?

Reply:
i.e. what is the best semantic to convey this as a drop target, and is it even useful for AT?

Reply:
Turns out there is a bit of Aria for this, although it is deprecated with no replacement. The `aria-grabbed` and `aria-dropeffect` are meant to indicate drag actions and drop targets, which can be any element in the base markup.  https://w3c.github.io/aria/#attrs_dragdrop

Reply:
Those would be out of scope for this component. I believe this is more of keeping this component clean to maybe I went a little far on making it perfect for it singular purpose. This component to me is just a image that is a circle with a line. :)

So I guess since this in the context of drag n drop which the name is very pointed at it will always be decorative so we don't really need the name. The accessibility for drag n drop will be handled not by this component at all.

Reply:
Yeah that was my understanding of the a11y for the d&d as well.

Mozilla describes the use of [role="img"](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Role_Img) differently here, but makes a point of suggesting it needs to be labelled, and if applied to an `svg` (that is not decorative), that it should be applied to the outer `svg`.

For this, I see the image as just an internal detail for the visual representation.  It could have been constructed with `div`s without affecting the meaning.  i.e. we're not surfacing this to AT with the expectation that it will get announced and described as an image.

I feel like I am making a big deal out of something that is small, but I don't agree with the semantics here.

Reply:
@dbatiste is there anything else that need to be done with this PR before owners approval ?

Reply:
The `role="img"` should be removed, since it does not define this group of svgs in an accessible way.  If it is used (which I do not think it should be in this case), the element needs to be labelled.